docs/UserGuide.md

```diff
@@ -713,6 +721,7 @@ A: Install the app in the other computer and overwrite the empty data file it cr
 | Address of student                                  | a/      | `a/15 Bukit Timah Road, Singapore 155203`            |
 | Amount paid by student                              | paid/   | `paid/300`                                           |
 | Amount owed by student                              | owed/   | `owed/200`                                           |
+| Rates per Class                                     | rate/   | `rate/100`                                           |
 | Additional notes of student (create new notes)      | nt/     | `nt/I have created new notes for Alice`              |
 | Additional notes of student (append existing notes) | nt-a/   | `nt-a/I have added more notes to the existing notes` |
 | Tags of student                                     | t       | `t/python`                                           |
```

Comment:
ps not your error but can you update the Tags of student to include a `/` behind the `t`?